libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/IncidentAlert.kt

```diff
@@ -0,0 +1,216 @@
+     * Indicates whether some other object is "equal to" this one.
+     */
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+        if (!super.equals(other)) return false
+
+        other as IncidentAlert
+
+        if (incidentType != other.incidentType) return false
+
+        return true
+    }
+
+    /**
+     * Returns a hash code value for the object.
+     */
+    override fun hashCode(): Int {
+        var result = super.hashCode()
+        result = 31 * result + incidentType
+        return result
+    }
+
+    /**
+     * Returns a string representation of the object.
+     */
+    override fun toString(): String {
+        return "IncidentAlert(incidentType=$incidentType), ${super.toString()}"
+    }
+
+    /**
+     * Use to create a new instance.
+     *
+     * @see IncidentAlert
+     */
+    class Builder(
+        private val coordinate: Point,
+        private val distance: Double
+    ) {
+        private var alertGeometry: RouteAlertGeometry? = null
+        private var incidentType: Int = IncidentAlertType.INCIDENT_UNKNOWN
+        private var incidentId: String? = null
+        @IncidentAlertImpact.Impact private var impact: String? = null
+        private var incidentCongestion: IncidentCongestion? = null
+        private var isClosed: Boolean = false
+        private var creationTime: Date? = null
+        private var startTime: Date? = null
+        private var endTime: Date? = null
+        private var description: String? = null
+        private var subType: String? = null
+        private var subTypeDescription: String? = null
+        private var alertcCodes: List<Int>? = null
+
+        /**
+         * Add optional geometry if the alert has a length.
+         */
+        fun alertGeometry(alertGeometry: RouteAlertGeometry?): Builder = apply {
+            this.alertGeometry = alertGeometry
+        }
+
+        /**
+         * Unique identifier for incident. Mustn't be **null**.
+         */
+        fun incidentId(incidentId: String): Builder = apply {
+            this.incidentId = incidentId
+        }
+
+        /**
+         * One of [IncidentAlertType].
+         */
+        fun incidentType(incidentType: Int): Builder = apply {
+            this.incidentType = incidentType
+        }
+
+        /**
+         * Severity level of incident.
+         */
+        fun impact(@IncidentAlertImpact.Impact impact: String?): Builder = apply {
+            this.impact = impact
+        }
+
+        /**
+         * Quantitative descriptor of congestion.
+         */
+        fun incidentCongestion(incidentCongestion: IncidentCongestion?): Builder = apply {
+            this.incidentCongestion = incidentCongestion
+        }
+
+        /**
+         * **True** if road is closed and no possibility to pass through there. **False**
+         * otherwise.
+         */
+        fun isClosed(isClosed: Boolean): Builder = apply {
```

Comment:
```suggestion
        fun isRoadClosed(isClosed: Boolean): Builder = apply {
```

Reply:
we have `closed` name in directions and native, I would like to keep `isClosed`(that is common for kotlin/java bool) or return just to `closed`